ui/src/components/CsvSourceMapping.vue

```diff
@@ -112,7 +112,9 @@ export default class CsvSourceMapping extends Vue {
   selectedColumns: CsvColumn[] = []
 
   get sourceTables (): Table[] {
-    return this.tables.filter(({ csvSource }) => csvSource?.id.equals(this.source.id))
+    return this.tables
+      .filter(({ csvSource }) => csvSource?.id.equals(this.source.id))
+      .sort(({ isObservationTable: o1 }, { isObservationTable: o2 }) => (o1 === o2) ? 0 : (o1 ? -1 : 1))
```

Comment:
Just wondering, should we sort the rest alphabetically or something?

Reply:
Or in the exact order they were created, with option to change that :)

Reply:
That would be great. But the really important thing visually is that observation tables appear first.

Reply:
![image](https://user-images.githubusercontent.com/588128/100261428-02258c80-2f4b-11eb-9959-169fad928b97.png)